src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entry.html.twig

```diff
@@ -147,6 +147,41 @@ main {
     padding: 0;
 }
 </style>
+
+<script type="text/javascript">
+
+var pageId = function () {
+    return {
+        beforeAnnotationCreated: function (ann) {
+            ann.entry = {{ entry.id }};
+        }
+    };
+};
+
+
+var app = new annotator.App();
+app.include(annotator.ui.main, {
+    element: document.querySelector('article')
+});
+app.include(annotator.storage.http, {
+    prefix: '/annotations',
+    urls: {
+        create: '/{{ entry.id }}',
```

Comment:
You should better generate each url using twig instead of _by hand_. It avoids to use a prod url in a dev environment (since your are calling `/annotations`)